FX/BryanChi_FX Devices/FXD Saike BandSplitter.jsfx

```diff
@@ -1,6 +1,12 @@
 noindex: true
 
-license: MIT
+desc:FXD Saike BandSplitter
+noindex: true
```

Comment:
This line is duplicated.
```suggestion
```

Reply:
my apologies, thanks for checking it for me!